docs/nddata/nddata.rst

```diff
@@ -171,3 +204,8 @@ array like this::
     >>> input_array = np.ma.array([1, 2, 3, 4], mask=[1, 0, 0, 1])
     >>> ndd_masked = NDData(input_array)
     >>> masked_array = np.ma.array(ndd_masked.data, mask=ndd_masked.mask)
+
+The same applies to extracting the data as Quantity::
+
+    >>> ndd_quantity = NDData([1, 2, 3, 4], unit="meter")
+    >>> quantity = u.Quantity(ndd_quantity, unit=ndd_quantity.unit)
```

Comment:
@bsipocz I forgot the ``.data`` while converting it. ``quantity = u.Quantity(ndd_quantity.unit, unit=ndd_quantity.unit)`` works.

Inside ``NDData`` the ``data`` and ``unit`` are seperated (that was one thing I did not change) so Quantity cannot recognize the unit by default.